base_sequence_option/__manifest__.py

```diff
@@ -4,13 +4,14 @@
 {
     "name": "Base Sequence Option",
     "summary": "Alternative sequence options for specific models",
-    "version": "14.0.1.0.0",
+    "version": "15.0.1.0.0",
     "author": "Ecosoft, Odoo Community Association (OCA)",
     "maintainers": ["kittiu"],
     "development_status": "Alpha",
     "website": "https://github.com/OCA/server-tools",
     "category": "Tools",
     "depends": ["base"],
+    "external_dependencies": {"python": ["odoo_test_helper"]},
```

Comment:
You don't need to add this here, just add it to test-requirements.txt (or create it if it does not exist :smile: )

Reply:
OK, line removed. The `test-requirements.txt` already contains the `odoo_test_helper` so didn't do anything there.